tests/kokkos/kokkos_utils_tests.cpp

```diff
@@ -194,4 +194,61 @@ TEST_CASE("team_utils_large_ni", "[kokkos_utils]")
   test_utils_large_ni(.5);
 }
 
+template<typename Scalar, int length, bool Serialize>
+void test_parallel_reduce()
+{
+  using Device = ekat::DefaultDevice;
+  using MemberType = typename ekat::KokkosTypes<Device>::MemberType;
+  using ExeSpace = typename ekat::KokkosTypes<Device>::ExeSpace;
+
+#ifdef KOKKOS_ENABLE_OPENMP
+  const int n = omp_get_max_threads();
+  // test will not work with more than 16 threads
```

Comment:
Why?

Reply:
I copy and pasted from an existing test above and this was part of it. I can remove it.

Reply:
Please do. Also, when copy-pasting code, always study the code closely, as otherwise poor assumptions and bugs will propagate.